modules/core/src/main/scala/scala/build/internals/OsLibc.scala

```diff
@@ -56,7 +56,7 @@ object OsLibc {
   // FIXME These values should be the default ones in coursier-jvm
 
   lazy val jvmIndexOs: String = {
-    val default = JvmChannel.defaultOs
+    val default = JvmIndex.defaultOs
```

Comment:
?

Reply:
Oh, thanks, didn't mean to commit this. I might have an incorrect java version or something and it wasn't compiling for me, so this was the fastest way to fix it locally. I'll revert this.